### PR TITLE
fix urn number normalize

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
Fixes https://textit.uservoice.com/admin/tickets/1879

File: temba/contacts/models.py [hunk 1]
```diff
@@ -1259,7 +1259,7 @@ def normalize_number(cls, number, country_code):
         number = regex.sub('[^0-9a-z\+]', '', number.lower(), regex.V0)
 
         # add on a plus if it looks like it could be a fully qualified number
-        if len(number) > 11 and number[0] != '+':
+        if len(number) >= 11 and number[0] != '+':
```
Comment (nicpottier) [inline temba/contacts/models.py]: I feel like we've changed this back and forth many times in the past.. haven't we?

Comment (norkans7) [inline temba/contacts/models.py]: I think so, though I saw https://github.com/nyaruka/rapidpro/blob/fix_urn_number_normalize/temba/contacts/models.py#L573 which remove the + on the value

Comment (nicpottier) [inline temba/contacts/models.py]: I guess as long as we keep adding unit tests to make sure we don't break past behavior then this is fine..

Comment (norkans7) [inline temba/contacts/models.py]: yep

**Pull request merged**